parser/worker.go

```diff
@@ -336,7 +340,11 @@ func (w Worker) ExportTxs(txs []*types.Tx) error {
 		}
 
 		// call the tx handlers
-		go w.handleTx(tx)
+		wg.Add(1)
+		go func(tx *types.Tx) {
+			defer wg.Done()
+			w.handleTx(tx)
```

Comment:
We are not logging the error that is returned by `handleTx`. Maybe is better to make sure `handleTx` does not return any error, and log it inside the function as it was initially?